src/lib/public-mappings/device-model-v1.ts

```diff
@@ -0,0 +1,53 @@
+import type { DeviceModelV1 } from "@seamapi/types/devicedb"
+
+import type { StoredDeviceModelV1, StoredManufacturer } from "lib/models.ts"
+
+import { publicMapImageReference } from "./image-reference.ts"
+import { publicMapManufacturer } from "./manufacturer.ts"
+
+interface PublicMapDeviceModelV1Options {
+  device_model: StoredDeviceModelV1
+  manufacturer: StoredManufacturer
+  fake_devicedb_base_url: string
+  external_image_proxy_endpoint?: string
```

Comment:
Still a bit unclear to me why these cannot be combined, when are `fake_devicedb_base_url` and `external_image_proxy_endpoint` ever used together?

Reply:
`fake_devicedb_base_url` is always required, `external_image_proxy_endpoint` is needed when loading images outside of the sample one

see https://github.com/seamapi/fake-devicedb/pull/4#discussion_r1357449240